Running in wrapper mode for '20327'

Running python job.

{u'nfeatures': array([48]), u'bow_size': array([1390]), u'svm_gamma': array([-2.73251149]), u'edgeThreshold': array([ 16.86884728]), u'svm_C': array([ 3.]), u'sigma': array([ 1.62299179]), u'contrastThreshold': array([ 0.001])}
----- Fold: 0
Testing with 951 pos 1473 neg
Training with 3349 pos 5490 neg
--- Gather features---
--- Train BOW---
--- Make hists---
--- Fit model---
/usr/local/lib/python2.7/dist-packages/sklearn/svm/base.py:176: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  (self.gamma == 'auto')):
/usr/local/lib/python2.7/dist-packages/sklearn/svm/base.py:179: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  elif self.gamma == 'auto':
--- Overall training model took 4075.71717405 ---
TP: 699 FP: 272 TN 1201 FN: 252
--- kfold: 0, accuracy: 0.783828382838, exp_mean*coverage: 0.555351501094
----- Fold: 1
Testing with 900 pos 1372 neg
Training with 3400 pos 5591 neg
--- Gather features---
--- Train BOW---
--- Make hists---
--- Fit model---
--- Overall training model took 4250.59540105 ---
TP: 666 FP: 186 TN 1186 FN: 234
--- kfold: 1, accuracy: 0.81514084507, exp_mean*coverage: 0.618304348244
----- Fold: 2
Testing with 785 pos 1569 neg
Training with 3515 pos 5394 neg
--- Gather features---
--- Train BOW---
--- Make hists---
--- Fit model---
--- Overall training model took 4132.55380607 ---
TP: 580 FP: 231 TN 1338 FN: 205
--- kfold: 2, accuracy: 0.814783347494, exp_mean*coverage: 0.603082601881
----- Fold: 3
Testing with 889 pos 1174 neg
Training with 3411 pos 5789 neg
--- Gather features---
--- Train BOW---
--- Make hists---
--- Fit model---
--- Overall training model took 4446.56301808 ---
TP: 714 FP: 146 TN 1028 FN: 175
--- kfold: 3, accuracy: 0.844401357247, exp_mean*coverage: 0.672459478034
----- Fold: 4
Testing with 775 pos 1375 neg
Training with 3525 pos 5588 neg
--- Gather features---
--- Train BOW---
----------------------------------------
Problem running the job:
(<type 'exceptions.SystemExit'>, SystemExit(0,), <traceback object at 0x7f03b0114098>)
Traceback (most recent call last):
  File "/home/hans/wart-detection/spearmint/spearmint/spearmint/runner.py", line 41, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/hans/wart-detection/spearmint/spearmint/spearmint/runner.py", line 131, in run_python_job
    result = module.main(job.id, params)
  File "/home/hans/wart-detection/bay_opt/experiment.py", line 39, in main
    kappa = cross_validate_with_participants(5, parts, dect_params=dect_params, bow_size=params['bow_size'], model_params=model_params)
  File "/home/hans/wart-detection/analyze_training.py", line 44, in cross_validate_with_participants
  File "/home/hans/wart-detection/analyze_training.py", line 125, in train_model
  File "/home/hans/wart-detection/analyze_training.py", line 184, in train_bagofwords
  File "/home/hans/wart-detection/spearmint/spearmint/bin/../spearmint/main.py", line 323, in sigint_handler
    sys.exit(0)
SystemExit: 0
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 21149.53 seconds.
